schemas/telemetry/crash/crash.4.schema.json

```diff
@@ -1574,6 +1578,10 @@
               "description": "1, Windows-only, present only if user32.dll was loaded before the DLL blocklist has been initialized",
               "type": "string"
             },
+            "UtilityActorsName": {
+              "description": "<string>, Comma-separated list of IPC actors name running on this Utility process instance",
+              "type": "string"
+            },
```

Comment:
also needs to be added here: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/d7c892dbfe46c37c44d897b8edb8b0d9f5273fec/templates/telemetry/crash/crash.4.schema.json#L212-L219